director/director.go

```diff
@@ -581,7 +581,7 @@ func registerServeAd(engineCtx context.Context, ctx *gin.Context, sType server_s
 		log.Warningf("Failed to parse server Web URL %v: %v\n", adV2.WebURL, err)
 		ctx.JSON(http.StatusBadRequest, server_structs.SimpleApiResp{
 			Status: server_structs.RespFailed,
-			Msg:    "Invalid server Web URL",
+			Msg:    fmt.Sprintf("Invalid %s registration. WebURL %s is not a valid URL", sType, adV2.WebURL),
```

Comment:
Same here, if we can point the user to the actual configuration knobs at their server they need to adjust to fix the invalid registration, we'll empower them to fix it themselves.